package-lock.json

```diff
@@ -1075,9 +1075,9 @@
       }
     },
     "@electron/get": {
-      "version": "1.10.0",
```

Comment:
are any of these dependency upgrades needed? if not, I'd suggest to not commit this file as having dep upgrades will need further review to ensure no breaking changes are introduced

Reply:
Ok, will revert it back. Probably not needed at all